docs/products/m3db/howto/monitoring.rst

```diff
@@ -0,0 +1,14 @@
+Monitor Aiven Services with M3DB
+================================
+
+M3DB is a perfect fit for a monitoring platform. It is designed to handle large volumes of metrics, and it's very good at that. Whether you are already working at scale or just curious about M3DB and monitoring, setting up an M3DB to monitor your existing Aiven services (or new ones if you like) is a really nice way to get started with this platform.
+
+Start from the service overview page of the service you would like to monitor with M3. So if you're going to track the metrics of your PostgreSQL service on Aiven, then start on the PostgreSQL service overview page.
```

Comment:
Should we put in bold **Service Overview** to make it more visible to readers?

Reply:
can do